src/gui_common/tooltip/ToolTipManager.tscn

```diff
@@ -75,6 +75,7 @@ DisplayDelay = 0.0
 [node name="speciesPreview" parent="GroupHolder/default" instance=ExtResource("3")]
 visible = false
 layout_mode = 0
+anchors_preset = 0
```

Comment:
Does this impact anything?

Reply:
Not as far as I can tell. I've removed it.